utils/cooperations/coopsAggregateOptions.js

```diff
@@ -0,0 +1,13 @@
+const coopsAggregateOptions = (params) => {
+  const { id } = params
+
+  let match = {}
+
+  if (id) {
```

Comment:
I guess this will look better, try it:  

const match = id
    ? { $or: [{ initiatorUserId: id }, { recipientUserId: id }] }
    : {}

Reply:
In Dmytro's pull request, there is also code related to this match object. Considering that, there is no need to rewrite it

Reply:
the result will be the same, in this example you don`t need to use let and it looks better